.renovaterc.json

```diff
@@ -10,5 +10,23 @@
     "group:typescript-eslintMonorepo"
   ],
   "reviewers": ["team:spindle-working-group"],
-  "prConcurrentLimit": 0
+  "prConcurrentLimit": 0,
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
+      "matchPackagePatterns": [
```

Comment:
`matchPackagePatterns` を使って部分一致したpackageを対象にします
https://docs.renovatebot.com/configuration-options/#matchpackagepatterns

Reply:
Q: extendしてるgroupたちとの兼ね合いどうなるんだっけか。うまく上書きして設定適用してくれるのかな。もしくはpackageRulesの方に設定移しちゃうとかもあるのかなぁ

https://github.com/openameba/spindle/blob/bec24ef1aa0e0fc9b92faa4100b00389c54d6b1c/.renovaterc.json#L7-L10

Reply:
> Important to know: Renovate will evaluate all packageRules and not stop once it gets a first match. Therefore, you should order your packageRules in order of importance so that later rules can override settings from earlier rules if necessary.
https://github.com/renovatebot/config-help/issues/684#issuecomment-640701387

後のルールが上書きする感じの挙動になるっぽい？ので問題なさそうな気がしました